app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -28,68 +21,64 @@ def create
           status: 'error',
           data:   @resource.as_json,
           errors: [I18n.t("devise_token_auth.registrations.missing_confirm_success_url")]
-        }, status: 403
+        }, status: :bad_request
```

Comment:
`bad_request` seems the most appropriate given the missing parameter. There is a good discussion on [stackoverflow](http://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data) about `400` vs `422`